mantis-control-plane/mantis-control-plane-core/src/main/java/io/mantisrx/server/master/resourcecluster/ResourceCluster.java

```diff
@@ -102,6 +102,15 @@ public interface ResourceCluster extends ResourceClusterGateway {
      */
     CompletableFuture<Ack> disableTaskExecutorsFor(Map<String, String> attributes, Instant expiry);
 
+    /**
+     * Enables/Disables scaler for a given skuID of a given clusterID
+     *
+     * @param skuID skuID whom scaler will be enabled/disabled.
+     * @param enabled whether the scaler will be enabled/disabled.
+     * @return a future that completes when the underlying operation is registered by the system
+     */
+    CompletableFuture<Ack> setScalerStatus(ContainerSkuID skuID, Boolean enabled);
```

Comment:
Is this interface the right place for this method? It feels like we will be delegating to the scaler actor here.